packages/default/scss/scrollview/_layout.scss

```diff
@@ -1,154 +1,175 @@
 
-$scrollview-pager-offset: 20px !default;
-$scrollview-pager-item-spacing: 20px !default;
-
 @include exports('scrollview/layout') {
-    $scrollview-page-button-size: 8px;
-    $scrollview-arrow-icon-size: 4.5em;
 
-    .k-scrollview-wrap {
+    //refactor when https://github.com/telerik/kendo-angular-scrollview/issues/130 is done
+    kendo-scrollview.k-scrollview-wrap,
+    div.k-scrollview {
         position: relative;
         overflow: hidden;
         outline: 0;
         display: block;
-    }
 
-    .k-scrollview {
-        list-style-type: none;
-        position: absolute;
-        margin: 0;
-        padding: 0;
-        width: 100%;
-        height: 100%;
-        cursor: default;
-
-        img {
-            user-select: none;
+        .k-scrollview,
+        .k-scrollview-wrap {
+            list-style-type: none;
+            position: absolute;
+            margin: 0;
+            padding: 0;
+            width: 100%;
+            height: 100%;
+            cursor: default;
+
+            img {
+                user-select: none;
+            }
+
+            & > li {
+                display: inline-block;
+                overflow: hidden;
+                position: absolute;
+                top: 0;
+                left: 0;
+            }
         }
 
-        & > li {
-            display: inline-block;
-            overflow: hidden;
+        .k-scrollview-pageable,
+        .k-scrollview-nav {
+            margin: 0 auto;
+            padding: 0;
+            width: 100%;
+            display: flex;
+            justify-content: center;
+            text-align: center;
+            list-style: none;
             position: absolute;
-            top: 0;
             left: 0;
+            bottom: $scrollview-pager-offset;
+            pointer-events: none;
         }
-    }
 
-    .k-pages,
-    .k-scrollview-pageable {
-        margin: 0 auto;
-        padding: 0;
-        width: 100%;
-        display: flex;
-        justify-content: center;
-        text-align: center;
-        list-style: none;
-        position: absolute;
-        left: 0;
-        bottom: $scrollview-pager-offset;
-        pointer-events: none;
-    }
-    .k-pages > li,
-    .k-scrollview-pageable > li.k-button {
-        margin: 0 $scrollview-pager-item-spacing;
-        padding: 0;
-        width: $scrollview-page-button-size;
-        height: $scrollview-page-button-size;
-        box-sizing: content-box;
-        vertical-align: middle;
-        display: inline-block;
-        flex: 0 0 $scrollview-page-button-size;
-        border-radius: 50%;
-        cursor: pointer;
-        pointer-events: all;
-    }
+        .k-scrollview-pageable > li.k-button,
+        .k-scrollview-nav > li.k-link {
+            margin: 0 $scrollview-pager-item-spacing;
+            padding: 0;
+            width: $scrollview-pagebutton-size;
+            height: $scrollview-pagebutton-size;
+            box-sizing: content-box;
+            vertical-align: middle;
```

Comment:
This align style is not applied here, move it for jquery rendering styles instead